### DISCUSSION TITLE
Remove duplicate words

### DISCUSSION THREAD

**Discussion by carlosmgilp** [microsoft/vscode-discussions#1138] [Extension Development QnA] [answered]
Hello how are you.
I wanted to ask if anyone knows of an extension for vscode that can be used to eliminate duplicate words in a selection.
I need to select some words, which can be on one or several lines and for the extension to eliminate all duplicate words in the selection.
Beforehand thank you very much.
Excuse my English, I only speak Spanish.

**Comment by alefragnani** [comment, marked as answer]
Hi @carlosmgilp ,

I made a quick search in the marketplace (https://marketplace.visualstudio.com/search?term=Remove%20Duplicate%20word&target=VSCode&category=All%20categories&sortBy=Relevance), but didn’t find any extension that does what you want. 

You could however, use the `Add Selection to Next Find Match` command, to select every occurrence of the desired word and delete them all with one simple `DEL`.

By the way, for future questions or feature requests about VS Code usage, please use the VS Code repository instead (https://github.com/microsoft/vscode/issues/new/choose). This Discussions repo is focused for extension development for now.

Hope this helps
